### PR TITLE
fix e2e tests

### PR DESCRIPTION
**Pull request by pashidlos**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ services:
       - postgres
   postgres:
     container_name: postgres
-    image: postgres:12-alpine3.18
+    image: postgres
```
Comment (paazmaya) [inline docker-compose.yml:31]: Why dropping alpine and versions? It does make sense to define them to know what version of Postgres is running and Alpine to keep images small...

Comment (pashidlos) [reply docker-compose.yml:31]: I had an issue running it with `pgAdmin`

Comment (paazmaya) [reply docker-compose.yml:31]: Could it be that there was an existing database data storage which was made in a different postgresql version?
At least I had those issues before defining a specific version...

Comment (paazmaya) [reply docker-compose.yml]: Added a commit to specify also the pgadmin version and link where to find the supported postres versions. Tested with fresh build and "works for me"

**Comment by paazmaya** [comment]
Related to pgadmin, I been curious about adding the server as preconfigured via json file https://www.pgadmin.org/docs/pgadmin4/latest/import_export_servers.html#json-format, but the password needs to be provided in a separate file using https://www.postgresql.org/docs/current/libpq-pgpass.html, and the use of that file was not working until recently https://github.com/pgadmin-org/pgadmin4/issues/6165...

**Comment by paazmaya** [comment]
Tried to server config with pgadmin version 4.7 (committed) and snapshot from few hours ago, but both have the same behaviour with not knowing the server password.
It is anyhow less work now when using pgadmin since only the password is needed, not all the server information.

In any case, this is @pashidlos your PR and you go forward with this as you want.

**Comment by pashidlos** [comment]
@paazmaya thanks, I'm good with it

**Pull request merged**